docs/source/documentation.rst

```diff
@@ -307,12 +310,124 @@ Then, use the following command::
     axondeepseg -t SEM -i test_segmentation/test_sem_image/image1_sem/
 
 Segment images from multiple folders
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To segment images that are located in different folders, specify the path to the folders in the **-i** argument, one after the other. For instance, to segment all the images of folders **'test_sem_image/image1_sem/'** and **'test_sem_image/image2_sem/'** of the test dataset, use the following command::
 
     axondeepseg -t SEM -i test_segmentation/test_sem_image/image1_sem/ test_segmentation/test_sem_image/image2_sem/
 
+
+Morphometrics
+---------------
+
+You can generate morphometrics using AxonDeepSeg via the command line interface.
+
+Syntax
+~~~~~
+
+The script to launch in called **axondeepseg_morphometrics**. It has several arguments.
+
+**Required arguments:**
+
+-i IMGPATH
+                    Path to the image file whose morphometrics needs to be calculated.
+
+**Optional arguments:**
+
+-s SIZEPIXEL        Pixel size of the image(s) to segment, in micrometers. 
+                    If no pixel size is specified, a **pixel_size_in_micrometer.txt** file needs to be added to the image folder path ( that file should contain a single float number corresponding to the resolution of the image, i.e. the pixel size). The pixel size in that file will be used for the segmentation.
+
+
+-a AXONSHAPE       Axon shape
+                    **circle:** Axon shape is considered as circle. In this case, diameter is computed using equivalent diameter. 
+                    **ellipse:** Axon shape is considered as an ellipse. In this case, diameter is computed using ellipse minor axis.
+                    The default axon shape is set to **circle**.
+
+-f FILENAME         Name of the excel file in which the morphometrics file will be stored.
+                    The excel file extension can either be **.xlsx** or **.csv**.
```

Comment:
The `csv` option is not working, see comment in `launch_morphometrics_computation.py`.

Reply:
Fixed this.